tests/pattern_rewriter_test.py

```diff
@@ -57,6 +57,60 @@ def match_and_rewrite(self, op: Operation, rewriter: PatternRewriter):
         PatternRewriteWalker(RewriteConst(), apply_recursively=False))
 
 
+def test_non_recursive_rewrite_f32():
```

Comment:
I do not actually see the value of having this here.
This file is about testing the pattern rewriter, so just changing the tests to use `i32` instead of `f32` won't really test anything new for the pattern rewriter here.
So I would suggest to instead just test the constructor in an `arith` specific test file.

Reply:
ok